assets/product-info.js

```diff
@@ -91,13 +93,34 @@ if (!customElements.get('product-info')) {
         this.#renderProductInfo(targetUrl, variant?.id, targetId, callback);
       }
 
+      // Load the product page and update sections
+      #updateProductSections(url, variantId) {
+        this.sectionRefreshAbortController?.abort();
+        this.sectionRefreshAbortController = new AbortController();
+        fetch(this.#getProductInfoUrl(url, variantId, true), {
+          signal: this.sectionRefreshAbortController.signal,
+        })
+          .then((response) => response.text())
+          .then((responseText) => {
+            const html = new DOMParser().parseFromString(responseText, 'text/html');
+            const updatedSections = Array.from(html.querySelectorAll('main > .shopify-section')).reduce(
```

Comment:
@tyleralsbury / @tauthomas01 It looked like selecting `shopify-section`s under `main` targets all sections that could contain a product metafield (and so would need to be updated). Is that correct?